### PR TITLE
Adjust loglevels in pulpcore-worker

### PR DESCRIPTION
**Pull request by mdellweg**
Also add traceback to logging on failed tasks.

[noissue]

### CODE REVIEW

File: pulpcore/tasking/pulpcore_worker.py [hunk 1]
```diff
@@ -208,14 +209,15 @@ def _perform_task(task_pk):
...
+                _logger.debug("Task is coroutine {}".format(task.pk))
                 loop = asyncio.get_event_loop()
                 loop.run_until_complete(result)
 
     except Exception:
         exc_type, exc, tb = sys.exc_info()
         task.set_failed(exc, tb)
-        _logger.info("Task failed {}".format(task.pk))
+        _logger.info("Task {} failed ({})".format(task.pk, exc))
+        _logger.info("\n".join(traceback.format_list(traceback.extract_tb(tb))))
```
Comment (mdellweg) [inline pulpcore/tasking/pulpcore_worker.py:220]: I would like to change this to "DEBUG", but it wouldn't show up anywhere.

My other thought is, we could introduce a `PulpTaskError` that reflects known cases to go wrong that loggs without a stacktrace. Thinking about "Manifest not found, check your remotes URL". There is no need to print 12 stack levels for that one.

**Review by dralley — APPROVED** [review]

File: pulpcore/tasking/pulpcore_worker.py (hunk 1 above)
Comment (bmbouter) [reply pulpcore/tasking/pulpcore_worker.py:220]: +1 to doing the logging like this in the future

Comment (bmbouter) [reply pulpcore/tasking/pulpcore_worker.py:220]: I tested this and the formatted TB is nice:

```
Jun 28 14:40:07 pulp3-source-fedora34.localhost.example.com pulpcore-worker[344050]: pulp [0ef5075628494830bfa2775b34d4b852]: pulpcore.tasking.pulpcore_worker:INFO:   File "/home/vagrant/devel/pulpcore/pulpcore/tasking/pulpcore_worker.py", line 210, in _perform_task
Jun 28 14:40:07 pulp3-source-fedora34.localhost.example.com pulpcore-worker[344050]:     result = func(*args, **kwargs)
Jun 28 14:40:07 pulp3-source-fedora34.localhost.example.com pulpcore-worker[344050]:   File "/home/vagrant/devel/pulp_file/pulp_file/app/tasks/synchronizing.py", line 42, in synchronize
Jun 28 14:40:07 pulp3-source-fedora34.localhost.example.com pulpcore-worker[344050]:     raise Exception('oh no')
```

File: pulpcore/tasking/pulpcore_worker.py [hunk 2]
```diff
@@ -115,7 +116,7 @@ def iter_tasks(self):
     def sleep(self):
         """Wait for signals on the wakeup channel while heart beating."""
 
-        _logger.info(f"Worker {self.name} entering sleep state.")
+        _logger.debug(f"Worker {self.name} entering sleep state.")
```
Comment (bmbouter) [inline pulpcore/tasking/pulpcore_worker.py:119]: :+1:

**Review by bmbouter — APPROVED** [review]
Thanks @mdellweg !

**Pull request merged**